app/controllers/users_controller.rb

```diff
@@ -0,0 +1,33 @@
+class UsersController < ApplicationController
+  def new
+    @user = User.new
+  end
+
+  def create
+    @user = User.new(user_params)
+
+    if @user.save
+      flash[:success] = 'Success'
+      redirect_to new_user_path
+    else
+      flash[:danger] = 'Try again'
+      render 'new'
+    end
+  end
+
+  def edit
+    @user = User.find(params[:id])
+  end
+
+  def update
+    @user = User.find(params[:id])
+    @user.update(user_params)
+    redirect_to edit_user_path(@user)
+  end
```

Comment:
Use the if construct to provide conditional output i.e. `redirect_to edit_user_path` if the user was updated or `render :edit` if the user was not updated.